src/dotnet/AzureAI.Proxy/Models/AzureOpenAIResponses/todo.txt

```diff
@@ -0,0 +1,7 @@
+Use these files to create classes to have a 
```

Comment:
Should we add this as an issue and remove the TODOs from this project?

Reply:
leave it for now please

Reply:
removed, no issue needed just yet